site/docs/pages/token/token-image.mdx

```diff
@@ -0,0 +1,110 @@
+import { TokenImage } from '../../../../src/token/components/TokenImage';
+import App from '../App';
+
+# `<TokenImage />`
+
+### Warning: component under development, may change in coming weeks
+
+The `TokenImage` component is an image that crops token image to a circle with an adjustable size.
+
+## Usage
+
+`TokenImage` with an url
+
+:::code-group
+
+```tsx [code]
+<TokenImage
+  src="https://dynamic-assets.coinbase.com/dbb4b4983bde81309ddab83eb598358eb44375b930b94687ebe38bc22e52c3b2125258ffb8477a5ef22e33d6bd72e32a506c391caa13af64c00e46613c3e5806/asset_icons/4113b082d21cc5fab17fc8f2d19fb996165bcce635e6900f7fc2d57c4ef33ae9.png"
+  size={16}
+/>
+
+<TokenImage
+  src="https://dynamic-assets.coinbase.com/dbb4b4983bde81309ddab83eb598358eb44375b930b94687ebe38bc22e52c3b2125258ffb8477a5ef22e33d6bd72e32a506c391caa13af64c00e46613c3e5806/asset_icons/4113b082d21cc5fab17fc8f2d19fb996165bcce635e6900f7fc2d57c4ef33ae9.png"
+  size={24}
+/>
+```
+
+```html [return html]
+<img
+  data-testid="ockTokenImage_Image"
+  src="https://dynamic-assets.coinbase.com/dbb4b4983bde81309ddab83eb598358eb44375b930b94687ebe38bc22e52c3b2125258ffb8477a5ef22e33d6bd72e32a506c391caa13af64c00e46613c3e5806/asset_icons/4113b082d21cc5fab17fc8f2d19fb996165bcce635e6900f7fc2d57c4ef33ae9.png"
+  style="width: 16px; height: 16px; border-radius: 50%; overflow: hidden; background: blue;"
+/>
+
+<img
+  data-testid="ockTokenImage_Image"
+  src="https://dynamic-assets.coinbase.com/dbb4b4983bde81309ddab83eb598358eb44375b930b94687ebe38bc22e52c3b2125258ffb8477a5ef22e33d6bd72e32a506c391caa13af64c00e46613c3e5806/asset_icons/4113b082d21cc5fab17fc8f2d19fb996165bcce635e6900f7fc2d57c4ef33ae9.png"
+  style="width: 24px; height: 24px; border-radius: 50%; overflow: hidden; background: blue;"
+/>
+```
+
+:::
+
+<App>
+  <div style={{ display: 'flex', gap: '8px', flexDirection: 'column'}}>
+    <TokenImage
+      src="https://dynamic-assets.coinbase.com/dbb4b4983bde81309ddab83eb598358eb44375b930b94687ebe38bc22e52c3b2125258ffb8477a5ef22e33d6bd72e32a506c391caa13af64c00e46613c3e5806/asset_icons/4113b082d21cc5fab17fc8f2d19fb996165bcce635e6900f7fc2d57c4ef33ae9.png"
+      size={16}
+    />
+
+    <TokenImage
+      src="https://dynamic-assets.coinbase.com/dbb4b4983bde81309ddab83eb598358eb44375b930b94687ebe38bc22e52c3b2125258ffb8477a5ef22e33d6bd72e32a506c391caa13af64c00e46613c3e5806/asset_icons/4113b082d21cc5fab17fc8f2d19fb996165bcce635e6900f7fc2d57c4ef33ae9.png"
+      size={24}
+    />
+
+  </div>
+</App>
+
+`TokenImage` with null as src
+
+:::code-group
+
+```tsx [code]
+<TokenImage
+  src="https://dynamic-assets.coinbase.com/dbb4b4983bde81309ddab83eb598358eb44375b930b94687ebe38bc22e52c3b2125258ffb8477a5ef22e33d6bd72e32a506c391caa13af64c00e46613c3e5806/asset_icons/4113b082d21cc5fab17fc8f2d19fb996165bcce635e6900f7fc2d57c4ef33ae9.png"
```

Comment:
Wait, where is the `null`? 

Also, we should point out to Shelley that we might need a more cute default image.

Also ideally we should give a way to customize the color....

I guess the custom bg, let's do that later, and see also how Sid think the theme story.